components/RoomTemplate.vue

```diff
@@ -30,6 +36,7 @@ const props = withDefaults(
   }>(),
   {
     issues: () => [],
+    estimationMethod: () => "Fibonacci",
```

Comment:
Shouldn't there also be a prop for this? Also "Fibonacci" does not seem to be a valid EstimationMethod. Do you mean "Story Points"?